actor/src/main/java/org/apache/pekko/japi/JAPI.java

```diff
@@ -15,8 +15,14 @@
 
 import scala.collection.Seq;
 
+@Deprecated
 public class JAPI {
 
+  /**
+   * @deprecated Use <code>Util.immutableSeq(java.util.Arrays.asList(...))</code> instead. Deprecated since
```

Comment:
I still don't know where this is Util class is. It could mean anything. It is too generic a name not to be fully qualified.